features/src/main/scala/com/salesforce/op/features/FeatureLike.scala

```diff
@@ -87,6 +82,26 @@ trait FeatureLike[O <: FeatureType] {
    */
   @transient protected lazy val log = LoggerFactory.getLogger(this.getClass)
 
+  /**
+   * The distribution information of the feature
+   * (is a sequence because map features have distribution for each key)
+   */
+  val distributions: Seq[FeatureDistributionLike]
+
+  /**
+   * The distribution information of the feature computed during training
+   * (is a sequence because map features have distribution for each key)
+   */
+  final def trainingDistributions: Seq[FeatureDistributionLike] =
```

Comment:
This is a bit weird to have as a sequence now since there should only be one training distribution for a feature, right? I'm okay with keeping the type as is for simplicity, but we should probably put it in the documentation if we expect it to always be a Seq of length one. Is this right, @leahmcguire ?